acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/HttpServiceLoginAcceptanceTest.java

```diff
@@ -27,39 +27,65 @@
 import org.junit.Test;
 
 public class HttpServiceLoginAcceptanceTest extends AcceptanceTestBase {
-
   private Cluster authenticatedCluster;
-  private BesuNode node;
+  private BesuNode nodeUsingAuthFile;
+  private BesuNode nodeUsingJwtPublicKey;
+
+  // token with payload{"iat": 1516239022,"exp": 4729363200,"permissions": ["net:peerCount"]}
+  private static final String TOKEN =
+      "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1MTYyMzkwMjIsImV4cCI6NDcyOTM2MzIwMCwicGVybWl"
+          + "zc2lvbnMiOlsibmV0OnBlZXJDb3VudCJdfQ.Y6mNV0nvjzOdqAgMgxknFAOUTKoeRAo4aifNgNrWtuXbJJgz6-"
+          + "H_0GvLgjlToohPiDZbBJXJJlgb4zzLLB-sRtFnGoPaMgz_d_6z958GjFD7x_Fl0HW-WrTjRNenZNfTyD86OEAf"
+          + "XHy-7N3OYY2a5yeDbppTJy6nnHTq9hY-ad22-oWL1RbK3T_hnUJII_uXCZ9bJggSfu5m-NNUrm3TeqdnQzIaIz"
+          + "DqHlL0wNZwVPB4cFGN7zKghReBpkRJ8OFlxexQ491Q5eSpuYquhef-yGCIaMfy7GVtpDSD3Y-hjOErr7gUNCUh"
+          + "1wlc3Rb7ru_0qNgCWTBPJeRK32GppYotwQ";
 
   @Before
   public void setUp() throws IOException, URISyntaxException {
     final ClusterConfiguration clusterConfiguration =
         new ClusterConfigurationBuilder().awaitPeerDiscovery(false).build();
-
     authenticatedCluster = new Cluster(clusterConfiguration, net);
-    node = besu.createArchiveNodeWithAuthentication("node1");
-    authenticatedCluster.start(node);
-    node.verify(login.awaitResponse("user", "badpassword"));
+
+    nodeUsingAuthFile = besu.createArchiveNodeWithAuthentication("node1");
+    nodeUsingJwtPublicKey = besu.createArchiveNodeWithAuthenticationUsingJwtPublicKey("node2");
+    authenticatedCluster.start(nodeUsingAuthFile, nodeUsingJwtPublicKey);
+
+    nodeUsingAuthFile.verify(login.awaitResponse("user", "badpassword"));
+    nodeUsingJwtPublicKey.verify(login.awaitResponse("user", "badpassword"));
   }
 
   @Test
   public void shouldFailLoginWithWrongCredentials() {
-    node.verify(login.failure("user", "badpassword"));
+    nodeUsingAuthFile.verify(login.failure("user", "badpassword"));
   }
 
   @Test
   public void shouldSucceedLoginWithCorrectCredentials() {
-    node.verify(login.success("user", "pegasys"));
+    nodeUsingAuthFile.verify(login.success("user", "pegasys"));
   }
 
   @Test
   public void jsonRpcMethodShouldSucceedWithAuthenticatedUserAndPermission() {
     final String token =
-        node.execute(permissioningTransactions.createSuccessfulLogin("user", "pegasys"));
-    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+        nodeUsingAuthFile.execute(
+            permissioningTransactions.createSuccessfulLogin("user", "pegasys"));
+    nodeUsingAuthFile.useAuthenticationTokenInHeaderForJsonRpc(token);
+
+    nodeUsingAuthFile.verify(net.awaitPeerCount(1));
+    nodeUsingAuthFile.verify(net.netVersionUnauthorizedExceptional("Unauthorized"));
```

Comment:
The test named with Success and ending with an `Unauthorized` check is a bit confusing. I believe I understand the idea that when using `netVersionUnauthorizedExceptional` we don't include the token, but reading the code it feels like the method name is wrong (even though it is also testing that the request succeeds with a request including the token.

Reply:
This should be two seperate tests. One testing that peer count works and another that net version fails. I've split them out now.